### ISSUE TITLE
Make Overview Page more compact

### ISSUE THREAD

**Issue by dillchen** [hicommonwealth/commonwealth#9700]
- Make Thread Previews take up less views
- Potentially stats

**Comment by sssssabinaaa** [comment]
Design here https://www.figma.com/design/dGPiswtnIfQIzSNOlB4MNC/Sticky-editor-%2B-Sidebar-%2B-Thread-UI?node-id=1282-6301&t=c3wKWWDermTRicLM-1

**Comment by KaleemNeslit** [comment]
@sssssabinaaa @Israellund  i have one question about new design 
-1 in previous design we have showing topic details in first column  in new design we are showing the thread Details in 1st columns
-2 in previous design we are showing topic and their thread in block  and in new design we are showing every thread without topic group 
 
![Image](https://github.com/user-attachments/assets/a07f1d61-8d2e-41b7-a7d5-39fae33a0ac6)
![Image](https://github.com/user-attachments/assets/2d0ff377-d4c6-42f5-8ab6-8f1d5c8abcd2)

**Comment by sssssabinaaa** [comment]
@dillchen afaik we are going with the second design correct?

**Comment by KaleemNeslit** [comment]
@dillchen i am working on filter in overView , i have a question 
in filter by topic we will show all topic as we are showing in imageView , when user select any topic we will fetch data from server against that topic ? 
because for this we need to update the backend

**Comment by dillchen** [comment]
Yes, we'll need to filter by topic, but we should have this data already as users can of course navigate to a specific topic (via the sidebar) already and view only posts from that topic

**Comment by KaleemNeslit** [comment]
here is the updated design @masvelio https://www.figma.com/design/D7NO95qj2COBpJcHTXoLXd/Community-Governance---Home?node-id=212-14660&t=zW6LBncfIknmnnno-0
